### PR TITLE
Fix tests for Travis CI to pass

### LINKED ISSUE #118: Update tests to pass Travis CI build
Note: **We are preparing to participate in Google Code-in, and have reserved this issue for participants in GCI - but we'd love to have your help with another one! Please check out https://code.publiclab.org to see more.**

### What's the problem (or idea)
The Travis CI build for the project fails due to the recent updates in the UI of the project. 

### What should be done?

You need to update these two test cases, to help pass the CI build.

https://github.com/publiclab/simple-data-grapher/blob/94bc33c2f5a8b26c155536e0c0ab2a66bd7797ae/spec/js/ui_tests.js#L30-L37

### Reference 

You can check the recent failed build log here: https://travis-ci.com/publiclab/simple-data-grapher/builds/141074484

Thank You 😊

### PR DESCRIPTION
**Pull request by Uzay-G**
Fixes #118

I fixed the tests. In one case there was a space on the title that shoulndn't have been there, and in the other there was an inconsistency between the sub-headings which I fixed.

You can check out the most recent functional build on Travis here: https://travis-ci.org/Uzay-G/simple-data-grapher/builds/627079233
Thanks!
What do you think @debck @publiclab/reviewers

### CODE REVIEW

File: spec/js/ui_tests.js [hunk 1]
```diff
@@ -29,11 +29,11 @@ describe("heading tests", function(){
     });
     it ("should test main heading", async function(){
         const headingValue = await page.$eval('.main_heading', el => el.innerHTML);
-        assert.equal(headingValue," Simple Data Grapher");
+        assert.equal(headingValue,"Simple Data Grapher");
     });
     it ("should test sub heading", async function(){
-        const headingValue = await page.$eval('.sub_heading', el => el.innerHTML);
-        assert.equal(headingValue,"Plot and Export Graphs with CSV data");
+        const headingValue = await page.$eval('.sub_title', el => el.innerHTML);
```
Comment (debck) [inline spec/js/ui_tests.js]: Was there any requirement for changing `.sub_heading` to `.sub_title`.

**Comment by Uzay-G** [comment]
Oh no sorry that's something left over from when I was trying to find the
error. I can change it back tonight if you'd like?

**Comment by debck** [comment]
Yes, you can change it..Also, run the build command and try running the tests once again before you push.  👍 Good work 
Thanks

**Comment by Uzay-G** [comment]
Alright @debck! I changed it back to sub_heading. You can check out the build for that commit here:
https://travis-ci.org/Uzay-G/simple-data-grapher/builds/627191993

**Pull request merged**